### PR TITLE
Update transparentContracting.yml

### PR DESCRIPTION
**Pull request by superroz**
reworded slightly. still waiting for useful link.

### CODE REVIEW

File: frameworks/digital-outcomes-and-specialists/questions/declaration/transparentContracting.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-question: Will you be transparent about your financial performance, in line with your contractual obligations and government guidance on Open Book Contract Management (OBCM)?
+question: Will you be transparent about your financial performance, and comply with your contractual obligations and government guidance on Open Book Contract Management (OBCM)?
```
Comment (quis) [inline frameworks/digital-outcomes-and-specialists/questions/declaration/transparentContracting.yml]: Do we need the acronym if we don't use it anywhere else?

Comment (superroz) [inline frameworks/digital-outcomes-and-specialists/questions/declaration/transparentContracting.yml]: Think we need to find out which is more commonly used before we get rid of it. Hoping Warren can clear up the confusion on this one.

Comment (quis) [inline frameworks/digital-outcomes-and-specialists/questions/declaration/transparentContracting.yml]: It's hard to find the "government guidance". Closest I can get is this, which mentions "open book … provisions" in the final point of the first document:
https://www.gov.uk/government/publications/commercial-capability-contract-management-standards

[gov.uk/search?q=obcm](https://www.gov.uk/search?q=obcm) and [google.co.uk/search?q="obcm"+site:gov.uk](https://www.google.co.uk/search?q="obcm"+site%3Agov.uk)
don't seem to produce anything.

**Comment by quis** [comment]
Closing in favour of #103

**Pull request closed without merging**